### ISSUE TITLE
Use multithreading to let sound play simultaneously as the timer runs

### ISSUE THREAD

**Issue by bliutwo** [bliutwo/pomodoro#1]
Right now, all timers pause during the time a sound is being played.

**Comment by bliutwo** [comment]
You probably want to do this:
```
For each time you make a sound,
    Make a new thread
    Play the sound
    End the thread after the sound is played
```

**Comment by bliutwo** [comment]
You probably want the original program on its own thread...? Maybe that's not how it works.

**Comment by bliutwo** [comment]
Fixed by 9095b58d7fe0c78d14c0f0b938bc0d91bd332a52
